components/advocates/AdvocateCard.vue

```diff
@@ -18,29 +18,29 @@
   </AppCard>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component, Prop } from 'vue-property-decorator'
+<script setup lang="ts">
+interface Props {
+  name: string,
+  image: string,
+  city: string,
+  country: string,
+  region: string,
+  slackId: string,
+  slackUsername: string
+}
 
-@Component
-export default class AdvocateCard extends Vue {
-  @Prop({ type: String, default: '' }) name!: string
-  @Prop({ type: String, default: '' }) image!: string
-  @Prop({ type: String, default: '' }) city!: string
-  @Prop({ type: String, default: '' }) country!: string
-  @Prop({ type: String, default: '' }) region!: string
-  @Prop({ type: String, default: '' }) slackId!: string
-  @Prop({ type: String, default: '' }) slackUsername!: string
+const props = withDefaults(defineProps<Props>(), {
+  city: '',
+  country: '',
+  region: '',
+  slackId: '',
+  slackUsername: ''
```

Comment:
These props are defined as mandatory on the interface, so they wouldn't need a default value. If they are optional, we should add `?` on the interface

Reply:
Made them optional in https://github.com/Qiskit/qiskit.org/pull/2935/commits/42cccafd1713be45e51d494226192a039192b8e9.